src/mlpack/core/data/tokenizer/char_split.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file char_split.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of CharSplit class which tokenizes string into character.
```

Comment:
into characters (I think the plural form suits better than the singular).

Reply:
Yeah.